sdk-lib/src/main/java/cash/z/ecc/android/sdk/SdkSynchronizer.kt

```diff
@@ -797,23 +779,23 @@ object DefaultSynchronizerFactory {
             initializer.rustBackend.cacheDbFile
         )
 
-    fun defaultService(initializer: Initializer): LightWalletService =
-        LightWalletGrpcService.new(initializer.context, initializer.lightWalletEndpoint)
+    fun defaultLightWalletClient(initializer: Initializer): BlockingLightWalletClient =
```

Comment:
I assume this maintains current functionality where a suspending function calls a blocking client. 
If we are planning to move to the coroutines implementation of the service, don't forget to create a github issue about it! :)

Reply:
https://github.com/zcash/zcash-android-wallet-sdk/issues/674